lib/megaLogger.js

```diff
@@ -94,27 +106,19 @@
      * @returns {boolean}
      * @private
      */
-    MegaLogger._environmentHaveSupportForColors = function() {
+    function _environmentHaveSupportForColors() {
         var isBrowser = true;
         var isIE = false;
 
-        if (typeof(window) === 'undefined') {
+        if (typeof(window) === 'undefined' || /PhantomJS/.test(window.navigator.userAgent)) {
             isBrowser = false;
         }
-        else if (typeof(window) !== 'undefined' && /PhantomJS/.test(window.navigator.userAgent)) {
-            isBrowser = false;
-        } else if (
-            typeof(window) !== 'undefined' && (
-            /MSIE/.test(window.navigator.userAgent) ||
-            /Trident/.test(window.navigator.userAgent) ||
-            /Edge/.test(window.navigator.userAgent)
-            )
-        ) {
+        else if ("ActiveXObject" in window) {
```

Comment:
I'm not a fan of implicit checks like this one, but ... i will merge it, it makes sense in terms of < RegExp/resources when checking.

Note: I will also add a comment to that line referring to this merge request.